host/xtest/Makefile

```diff
@@ -107,6 +107,7 @@ CFLAGS += -I../supp_plugin/include
 CFLAGS += -I$(out-dir)/xtest
 
 CFLAGS += -I$(OPTEE_CLIENT_EXPORT)/include
+CFLAGS += -I$(OPENSSL_DIR)/include
```

Comment:
What if `$(OPENSSL_DIR)` is an empty string? The compiler might warn about a non-existing include directory.

Reply:
yes, it is throwing errors, :(

Reply:
I have removed including the directory.  I can pass the directory through `CPATH` variable while calling `make`

Reply:
I assume you mean `CFLAGS`, what's stopping you from doing the same thing with `LDFLAGS`?

Reply:
To be precise, I cannot use `CFLAGS` and `LDFLAGS` while calling make as that will override those variables used in the xtest makefile.  If I want to append the LDFLAGS in the xtest Makefile with the LDFLAGS passed during make, I have to `override` the LDFLAGS in xtest Makefile.

For info: https://www.gnu.org/software/make/manual/html_node/Override-Directive.html

I have made those changes as well for reference. https://github.com/davidson-k/optee_test/commit/56bb3d3bbc5773ff5d3a361cb9f909684900774b
 
I'm ok with either of these two solution.

Reply:
gentle ping @jenswi-linaro  :)